assets/js/modules/analytics/datastore/__fixtures__/dashboard-user-dimensions-args.json

```diff
@@ -0,0 +1,59 @@
+{
+	"ga:channelGrouping": {
+		"startDate": "2020-12-09",
+		"endDate": "2021-01-05",
+		"compareStartDate": "2020-11-11",
+		"compareEndDate": "2020-12-08",
```

Comment:
These dates will only work for a specific `referenceDate`, so that needs to be passed in the Storybook story. In combination with my comment above (the offset of `1`), the dates here will only work with a reference date of `2021-01-06`. Either way, we'll need to pass `referenceDate`, otherwise the story will break tomorrow :)